persistence/src/commonMain/kotlin/com/wire/kalium/persistence/db/UserDatabaseBuilder.kt

```diff
@@ -60,6 +63,9 @@ class UserDatabaseBuilder internal constructor(
     private val platformDatabaseData: PlatformDatabaseData
 ) {
 
+    @OptIn(ExperimentalCoroutinesApi::class)
+    private val queriesContext: CoroutineContext = KaliumDispatcherImpl.io.limitedParallelism(4)
```

Comment:
Why `4`?


Also, for testing purposes, we'll need to inject this dispatcher in the constructor.

Reply:
> Why 4? 

i am still playing around with the configuration and see if it will affect performance 
for the time being i will remove the limitation

Reply:
still not done and i need to do the same changes for global DB